### ISSUE TITLE
Wrong autocomplete on Emacs with company-go-show-annotation turned on

### ISSUE THREAD

**Issue by dgiagio** [nsf/gocode#266]
The linked picture better describes the problem. Basically the autocomplete with company-go-show-annotation turned on duplicates the results on the same line concatenating the result with the annotation. I believe the correct is to output only the annotation, which already includes the method name and more.
![](http://i.imgur.com/DQuplmY.png)

**Comment by nsf** [comment]
I haven't forgotten about the issue, but holydays and stuff. :D Few more days...

**Comment by nsf** [comment]
So, I took a look at the problem and here's what I did.

I wanted to retain the existing behavior when `company-tooltip-align-annotations` set to `t`, but at the same time solve your problem. And now, when `company-tooltip-align-annotations` **is not** set to `t`, the annotations look this way:

![ann](https://cloud.githubusercontent.com/assets/12567/5690488/4f6b97ee-98ae-11e4-990e-270c87726f60.png)

I hope it works for you. Commit follows shortly.
